metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/ObjectsImporter.java

```diff
@@ -55,17 +53,11 @@
 
 public abstract class ObjectsImporter {
 
-    public static final String ASSEMBLY_END_TAG = "</ASSEMBLY>";
-
-    public static final String ENTRY_END_TAG = "</entry>";
-
     private static final Logger IMPORT_LOGGER = Logger.getLogger(ObjectsImporter.class.getName());
 
-    // XML retrievers. First is used as default, second for cases where API retrieval mode must be forced
+    // XML retrievers: first is used for all but reference sequence, and the second one only for reference sequences
     protected SraXmlRetrieverByAccession sraXmlRetrieverByAccession;
 
-    protected SraXmlRetrieverThroughApi sraxmlRetrieverByAccessionForceApi;
-
     protected AssemblyXmlRetrieverThroughEntrezApi assemblyXmlRetrieverThroughEntrezApi;
```

Comment:
Can we change the name to ReferenceSequenceXmlRetrieverThroughEntrezApi ??

Reply:
Sure thing. Fixed in https://github.com/EBIvariation/metadata-ws/pull/72/commits/14bfdda9b58a378990b84020cf273a89ba457795